package.json

```
@@ -12,7 +12,9 @@
     "lint": "eslint .",
     "prepare": "husky install",
     "prettier": "prettier --check '**/*.{js,ts,jsx,tsx,scss}'",
-    "prettier:write": "prettier --write '**/*.{js,ts,jsx,tsx,scss}'"
+    "prettier:write": "prettier --write '**/*.{js,ts,jsx,tsx,scss}'",
+    "playwright:install": "playwright install --with-deps",
+    "test": "playwright test"
```

We'll also need unit tests at some point as well, so let's make this
```suggestion
    "test:integration": "playwright test"
```
Since it'll be testing w3i like an integration test would, end to end.